src/js/disability-benefits/526EZ/config/form.js

```diff
@@ -121,17 +123,39 @@ const formConfig = {
         }
       }
     },
-    chapterThree: {
-      title: 'Chapter Three',
+    ratedDisabilities: {
+      title: 'Your Rated Disabilities',
       pages: {
-        pageOne: {
-          title: 'Page One',
-          path: 'chapter-three/page-one',
-          uiSchema: {},
+        ratedDisabilities: {
+          title: 'Your Rated Disabilities',
+          path: 'select-disabilities',
+          uiSchema: {
+            'view:selectDisabilitiesDescription': {
+              'ui:description': 'Please check all the disabilities that you’re filing an increase for because the condition has gotten worse.'
```

Comment:
You could probably just make this the description on `uiSchema`, instead of nesting it in this empty field.

Reply:
Ohh, right. I tried putting it on the `disabilities` array, but that didn't work (for what are now face-palmy obvious reasons).